client/verta/tests/unit_tests/deployment/test_build_scan.py

```diff
@@ -50,3 +50,23 @@ def test_get_scan(mock_conn, mocked_responses, build_dict, build_scan_dict):
         build_scan = build.get_scan()
 
     assert_build_scan_fields(build_scan, build_scan_dict)
+
+@settings(suppress_health_check=[HealthCheck.function_scoped_fixture])
+@given(build_dict=build_dict(), build_scan_dict=build_scan_dict())
+def test_start_scan(mock_conn, mocked_responses, build_dict, build_scan_dict):
```

Comment:
Since this test seems to explicitly be for external scans, we might want to call that out.

```suggestion
def test_start_external_scan(mock_conn, mocked_responses, build_dict, build_scan_dict):
```

Reply:
Done